src/anemoi/training/config/training/default.yaml

```diff
@@ -46,10 +46,12 @@ rollout:
   # maximum rollout to use
   max: 1
 
-max_epochs: 200
+max_epochs: null
+max_steps: 150000 # set to -1 to run for max_epochs
```

Comment:
Mix between null and -1, could this be made consistent?

Reply:
Context: pl.Trainer takes max_steps=-1  and max_epochs=None to turn them off. -

Done in 166fc130a94fb5318a1dfae0b9eea8989bdcb11d

New logic:

- config takes null for max_steps and max_epochs
- code to convert null value to -1 in train.py `max_steps = self.config.training.max_steps or